Mention installation gotchas (#32)
api3dao/ois: README.md

`npm install @api3/ois --save`

### Gotchas

#### Use pinned version

> We recommend using a pinned version of OIS package because the validation included in the package enforces the OIS

Reviewer: ```suggestion
We recommend using a pinned version of the OIS package because the validation included in the package enforces the OIS
```

> version up to the patch. This means if you use the validation to enforce validity of your OIS files, pinning the version
> will ensure your project uses the correct OIS version even if there are other packages depending on a different OIS
> version.

Reviewer: The first part of the sentence is a bit confusing to me, what about:
```suggestion
version up to the patch. Pinning the version also ensures your project uses the correct OIS version even if there are
other packages depending on a different OIS version.
```

Author: Yeah, much better.
